orchestra/google/marketing_platform/operators/display_video_360.py

```diff
@@ -460,27 +461,25 @@ def execute(self, context):
             advertisers = {}
             headers = False
             with open(report_file.name, 'r') as f:
-                for line in f:
-                    if not headers:
-                        headers = line
-                        headers = headers.strip().split(',')
-                        advertiser_id_index = headers.index('Advertiser ID')
-                        partner_id_index = headers.index('Partner ID')
-                    elif line.strip():
-                        line = line.strip().split(',')
-                        advertiser_id = line[advertiser_id_index]
-                        partner_id = line[partner_id_index]
-                        if advertiser_id.strip():
-                            try:
-                                advertisers[partner_id].append(advertiser_id)
-                                message = 'ADDING to key %s new advertiser %s' % (
-                                    partner_id, advertiser_id)
-                                logger.info(message)
-                            except KeyError:
-                                advertisers[partner_id] = [advertiser_id]
-                                message = 'CREATING new key %s with advertiser %s' % (
-                                    partner_id, advertiser_id)
-                                logger.info(message)
+                csv_reader = csv.reader(f)
```

Comment:
A csv.DictReader could be used instead, passing `fieldnames=["Advertiser ID", "Partner ID"]`. This way you can then do `advertiser_id = line["Advertiser ID"]` directly and remove lines [465-467].

Reply:
Hi @ceoloide, 

I have updated the file to use DictReader instead of reader. I found that when I passed the fieldnames parameter as you suggested that it would pick the first two fields in the CSV (Advertiser, Advertiser ID), and assign those to Advertiser ID and Partner ID respectively.

This could be because I'm using Python 2 in my Composer instance and the DictReader behaviour is subtly different between versions. So I left that bit out and I just access the fields from the full set.